ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcarousel/v1/productcarousel/clientlibs/js/product-carousel.js

```diff
@@ -12,130 +12,24 @@
  *
  ******************************************************************************/
 
+import Carousel from '../../../../../carousel/v1/carousel/clientlibs/js/carousel';
```

Comment:
Is this `import` really necessary? The general carousel clientlib exposes the `Carousel` class already globally, so it should be available, since in the archetypes all clientlibs are bundled together.

Reply:
The product carousel gets broken without the import.